py-polars/polars/series/series.py

```diff
@@ -6931,6 +6841,113 @@ def shift_and_fill(
 
         """
 
+    @deprecate_function("Use `Series.dtype.is_float()` instead.", version="0.19.13")
```

Comment:
```suggestion
    @deprecate_function("Use `Series.dtype.is_float()` instead.", version="0.19.14")
```
According to https://github.com/pola-rs/polars/releases/tag/py-0.19.14.

Reply:
For info, not for action:
All your deprecations force downstream libs sooner or later to do some maintenance work. (That’s ok). For my own lib, I‘d like to be free of deprecation warnings.

I did not find a deprecation on `is_float` in the release notes. So I looked in the code (with git blame) and found this line. From there on, I was able to track the correct release.